packages/x-data-grid/src/models/props/DataGridProps.ts

```diff
@@ -417,6 +417,10 @@ export interface DataGridPropsWithoutDefaultValue<R extends GridValidRowModel =
    * Override the height/width of the Data Grid inner scrollbar.
    */
   scrollbarSize?: number;
+  /**
+   * Override the height/width of the header filters.
+   */
+  filterHeaderHeight?: number;
```

Comment:
I am not against `filterHeaderHeight`, anyhow `headerFilterHeight` seems more aligned with the feature naming and sounds naturally more discoverable to me.
```suggestion
  headerFilterHeight?: number;
```